src/action.js

```diff
@@ -68,6 +70,8 @@ async function runCodeseeMapUpload(config, origin) {
     `https://github.com/${origin}`,
     "-a",
     config.apiToken,
+    "-b",
+    config.githubRef,
```

Comment:
I'm assuming there is a related codesee-cli change for this?

Reply:
https://github.com/Codesee-io/codesee/pull/788